Meshtastic/Helpers/BatteryHelper.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  BatteryHelper.swift
+//  Meshtastic
+//
+//  Created by Gabe Kangas on 8/6/24.
+//
+
+import Foundation
+import MeshtasticProtobufs
+
+class BatteryHelper {
```

Comment:
this can be an `enum`, since we won't ever need to instantiate an instance of `BatteryHelper`